Owncloud iOs Client/Login/Login/UniversalLoginViewController.swift

```diff
@@ -578,44 +670,143 @@ connection_declined  Connection declined by user
             
             self.setPasswordFooterError(message: NSLocalizedString("credentials_different_user", comment: "") )
         }
-        
-        
-        
     }
     
+    
 //MARK: Manage network errors delegate
     public func showError(_ message: String!) {
         DispatchQueue.main.async {
             self.setURLFooter(message: message, isType: .ErrorNotPossibleConnectToServer)
+            if !self.basicAuthInfoStackView.isHidden {
```

Comment:
As said offline: this is not to stay here, was a temporary patch until task of error review was done. And that's exactly what I'm doing now.

Thanks.